src/v/kafka/protocol/offset_fetch.h

```diff
@@ -62,7 +63,8 @@ struct offset_fetch_response final {
         data.error_code = error_code::none;
         if (topics) {
             for (auto& topic : *topics) {
-                std::vector<offset_fetch_response_partition> partitions;
+                small_fragment_vector<offset_fetch_response_partition>
```

Comment:
question: I'm probably missing some context here, but do we have a sense of what the "usual" size of this vector might be? As in, whether the `small_fragment_vector` fragment size (1KiB I believe) is the best/safest choice.

Reply:
Yeah large_fragment_vector is the other option, which is close to 128KiB and probably fine? I'm not really sure to be honest. I'll switch

Reply:
`fetchable_partition_response` is also small? So I guess it's expected to stay small because it's only data from this shard I believe?

Reply:
ya, i don't really know. I assume you could use `fragmented_vector` with any arbitrary fragment size if you were so inclined (rather than one of the convenience typedefs), but it's not clear to me whether there's any special reason to do so.

Certainly the 1KiB fragment size should prevent `bad_alloc` in all cases, so I don't want to hold this up. Just curious mostly.